### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Anu082000**
corrected few grammatical and linguistic errors

I understand that:

- [ ] I'm submitting this PR for reference only. It shows an example of what I'd like to see changed but
  I understand that it will not be merged and I will not be listed as a contributor on this project.

### CODE REVIEW

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for this PR

As a non-native speaker, I'll pass this to @robmadole and @talbs 

Will add a comment to a sentence for a clarification

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ a new CDN.
 Not familiar with Font Awesome 5? [Learn
 more](https://www.kickstarter.com/projects/232193852/font-awesome-5) about our
 successful Kickstarter and plan. You can also **[order Font Awesome
-Pro](https://fontawesome.com/pro)** which includes tons more icons directly
+Pro](https://fontawesome.com/pro)** which includes tons of  icons directly
```
Comment (tagliala) [inline README.md:13]: I think this means that FA Pro has more icons than FA Free.

Does changing `more` to `of` give the same idea?

**Review by anshulhub — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -105,8 +105,8 @@ we deviate from normal SemVer in the following ways:**
 
 ## License
 
-Font Awesome Free is free, open source, and GPL friendly. You can use it for
-commercial projects, open source projects, or really almost whatever you want.
+Font Awesome Free is a free, open-source, and GPL friendly. You can use it for
```
Comment (robmadole) [inline README.md:108]: Using the article `a` here turns `free, open-source, and GPL friendly` into adjectives without a subject.

File: README.md (hunk 1 above)
Comment (robmadole) [reply README.md:13]: @Anu082000 this changes the meaning. Was there something confusing about the original sentence?

**Pull request closed without merging**